ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.18.5",
+    "version": "0.19.0",
```

Comment:
I don't think you have to do a "breaking" change anymore, right? Also I published 0.18.6 just a bit ago - so you'd have to do 0.18.7

Reply:
![okeydokey](https://ih0.redbubble.net/image.251674867.2240/ap,550x550,12x16,1,transparent,t.u2.png)